test/scribe-plugin-content-cleaner.spec.js

```diff
@@ -7,19 +7,48 @@ var expect = chai.expect;
 var when = helpers.when;
```

Comment:
I think it would be nice to have unit tests for the transforms applied to a string rather than testing everything through the browser.